classes/player.rb

```diff
@@ -0,0 +1,26 @@
+class Player
+  attr_reader :name, :symbol, :log, :winner
+
+  def initialize (name, symbol)
+    @name = name
+    @symbol = symbol
+
+    @log = []
```

Comment:
Usually arrays, since they are collections of things, are named in plural
```suggestion
    @logs = []
```